includes/wpmenucart-settings.php

```diff
@@ -390,11 +390,11 @@ public function wpmenucart_options_do_page() {
 
 			</form>
 			<script type="text/javascript">
-			jQuery('.hidden-input').click(function() {
+			jQuery('.hidden-input').on('click', function() {
```

Comment:
This part of the code could use a bit of a makeover similar to what I suggested here:
https://github.com/wpovernight/woocommerce-improved-external-products/issues/2#issuecomment-815171252
(the repeated use of `jQuery` is unnecessary and those event triggers should ideally wait for jQuery to be loaded for sure)

Reply:
Like this? https://github.com/wpovernight/wp-menu-cart/pull/15/commits/a79ea5ad82f198c9a639c4cd87ba6aef4b795d74